.devcontainer/codespaces-docs/welcome.rst

```diff
@@ -1,5 +1,60 @@
-Welcome to GitHub PyMAPDL Codespaces for Documentalist!!
-========================================================
 
-This codespace has been specifically set for documenting PyMAPDL.
+===========================================================
+Welcome to GitHub PyMAPDL Codespaces for Documentalist!! 📚
+===========================================================
 
+
+🛑 Disclaimer
+=============
+
+You **must use this Codespace only for developing, contributing, documenting and/or
+write examples for PyMAPDL**.
+
+Additionally, this Codespace is using an MAPDL Student version which is
+available `here <https://www.ansys.com/academic/students>`_
+This version has some limitations.
+For more information visit
+`Ansys MAPDL Student <https://www.ansys.com/academic/students/ansys-student>`_
+
+
+📖 Codespace configuration
+==========================
+
+This Codespace has been set for helping developers to document and write examples
+in PyMAPDL in the easiest way. This Codespace contains all the OS and Python dependencies
+for running PyMAPDL and build the documentation in HTML and PDF.
+
+Some of the available Python dependencies are:
+
+* Numpy
+* Pandas
+* Pyvista
+* Sphinx
+* plotly
+
+
+🧐 How to
+=========
+
+To start documenting PyMAPDL, it is recommended you visit
+`Write documentation <https://mapdl.docs.pyansys.com/version/dev/getting_started/write_documentation.html>`_
+where the steps needed are detailed.
+
+You can see the latest documentation regarding using Codespaces with PyMAPDL
+`here <https://mapdl.docs.pyansys.com/version/dev/getting_started/devcontainer_link.html>`_.
+
+
+😊 Finally
+==========
+
+We hope you enjoyed this Codespace! 
```

Comment:
```suggestion
We hope you enjoy this Codespace. 
```
I'll let the exclamation point in line 53 remain.

Reply:
I appreciate a lot the concession! :P xD